aioesphomeapi/model.py

```diff
@@ -245,6 +255,13 @@ def _convert_climate_swing_modes(value):
     return [ClimateSwingMode(val) for val in value]
 
 
+def _convert_climate_presets(value):
+    return [ClimatePreset(val) for val in value]
+
+
+def _convert_string(value):
+    return [val for val in value]
```

Comment:
@glmnet lint complains about this line, however without this, HA fails to persist the object as JSON, as this object returned from the protobuf protocol is not a list, but some kind of google iterable, so here we just convert the iterable to a standard python list, so JSON serializer can work with it. How we should solve this lint check?

Reply:
https://stackoverflow.com/questions/28829236/is-it-possible-to-ignore-one-single-specific-line-with-pylint/29562085